log/README.md

```diff
@@ -21,6 +22,12 @@ func main() {
 }
 ```
 
+The log lines are printed in tuples separated with by a `=` symbol.
+
+```
+question=what is the meaning of life? answer=42
```

Comment:
Small note, I think that if a string contains spaces logfmt will [ensure it's quoted](http://godoc.org/github.com/kr/logfmt).